src/Wedding/Application/Http/Action/Web/Rsvp.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Wedding\Application\Http\Action\Web;
+
+use Symfony\Component\HttpFoundation\File\Exception\AccessDeniedException;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Twig\Environment;
+use Wedding\Application\Query\GetRsvpForInviteReference;
+use Wedding\Port\MessageBus\QueryBusInterface;
+
+class Rsvp
+{
+    private Environment $twig;
+    private TokenStorageInterface $tokenStorage;
+    private QueryBusInterface $queryBus;
+
+    public function __construct(Environment $twig, TokenStorageInterface $tokenStorage, QueryBusInterface $queryBus)
+    {
+        $this->twig = $twig;
+        $this->tokenStorage = $tokenStorage;
+        $this->queryBus = $queryBus;
+    }
+
+    public function __invoke(): Response
+    {
+        $token = $this->tokenStorage->getToken();
+
+        if (!$token instanceof TokenInterface) {
+            throw new AccessDeniedException();
+        }
+
+        $reference = $token->getUsername();
```

Comment:
Perhaps better using a ArgumentResolver to obtain a Invite object 🤔 less exposure to Symfony and less of a "god" service that plucks an authed user out